src/metadata.rs

```diff
@@ -48,7 +50,7 @@ impl FinalfusionApp for MetadataApp {
         if let Some(metadata) = read_metadata(&self.input_filename) {
             writer
                 .write_all(
-                    to_string_pretty(&metadata.0)
+                    to_string_pretty(&metadata.deref())
```

Comment:
did you try

```
to_string_pretty(&metadata)
```

? In some cases deref coercion kicks in.

Reply:
Yeah I tried, but ```serde::ser::Serialize``` is not implemented for ```Metadata```, so it doesn't work :(

Reply:
Ok, then you could use:

```
to_string_pretty(&*metadata)
```